charts/wordpress/v9.0.3/questions.yml

```diff
@@ -19,7 +19,7 @@ questions:
     type: string
     label: WordPress Image Name
   - variable: image.tag
-    default: "5.2.3-debian-9-r5"
+    default: "5.3.2-debian-10-r43"
```

Comment:
update the repo name to `bitnami/wordpress`

Reply:
This was updated in the last commit>